mysql-test/suite/wsrep/r/wsrep_set_var_extension.result

```diff
@@ -26,7 +26,7 @@ CALL test_hint("SET_VAR(wsrep_sync_wait=4)", "wsrep_sync_wait");
 hint_str
 SET_VAR(wsrep_sync_wait=4)
 @before_val	@hint_val	@after_val
-15	4	15
+0	4	0
 SET wsrep_sync_wait=2;
```

Comment:
@venkatesh-prasad-v why did this test result change?

Reply:
@satya-bodapati  I recorded this test while working on https://jira.percona.com/browse/PXC-3743. Back then `wsrep_sync_wait=15` was in the `mysql-test/suite/wsrep/my.cnf`. But I forgot to re-record it we I removed the line. So this change was necessary to make the test suite clean.